azure-pipelines-integration.yml

```diff
@@ -29,43 +29,53 @@ jobs:
         _repoName: "dotnet/roslyn"
         _targetSolution: "Compilers.sln"
         _branchName: "main"
-        _sha: "b670821b5150bc7ddb01f415d91c17dd55c752a2"
-      sdk:
-        _repo: "https://github.com/dotnet/sdk"
-        _repoName: "dotnet/sdk"
-        _targetSolution: "sdk.sln"
-        _branchName: "main"
-        _sha: "d32c8e7d19f3fc13190ff14cd960370a5d148995"
+        _sha: "a6a78dd64fc1eb466113ece272747a72d497aad7"
+
+      # TODO: Re-enable when formatting issue are investigated.
```

Comment:
Opened an issue for the disabled integration tests: https://github.com/dotnet/format/issues/1813